doc/rule-descriptions.md

```diff
@@ -12,62 +12,62 @@
-| [marquee](https://dequeuniversity.com/rules/axe/4.6/marquee?application=RuleDescription)                                         | Ensures &lt;marquee&gt; elements are not used                                                                                                         | Serious                  | cat.parsing, wcag2a, wcag222                                                      | failure                    |                                                                                                                                                                                                                                                                    |
-| [meta-refresh](https://dequeuniversity.com/rules/axe/4.6/meta-refresh?application=RuleDescription)                               | Ensures &lt;meta http-equiv=&quot;refresh&quot;&gt; is not used for delayed refresh                                                                   | Critical                 | cat.time-and-media, wcag2a, wcag221                                               | failure                    | [bc659a](https://act-rules.github.io/rules/bc659a), [bisz58](https://act-rules.github.io/rules/bisz58)                                                                                                                                                             |
-| [meta-viewport](https://dequeuniversity.com/rules/axe/4.6/meta-viewport?application=RuleDescription)                             | Ensures &lt;meta name=&quot;viewport&quot;&gt; does not disable text scaling and zooming                                                              | Critical                 | cat.sensory-and-visual-cues, wcag2aa, wcag144, ACT                                | failure                    | [b4f0c3](https://act-rules.github.io/rules/b4f0c3)                                                                                                                                                                                                                 |
-| [nested-interactive](https://dequeuniversity.com/rules/axe/4.6/nested-interactive?application=RuleDescription)                   | Ensures interactive controls are not nested as they are not always announced by screen readers or can cause focus problems for assistive technologies | Serious                  | cat.keyboard, wcag2a, wcag412                                                     | failure, needs&nbsp;review | [307n5z](https://act-rules.github.io/rules/307n5z)                                                                                                                                                                                                                 |
```

Comment:
4.a I think. You could argue that a scrollbar is functionality that needs to be keyboard accessibility.